eng/pipelines/aggregate-reports.yml

```diff
@@ -1,6 +1,67 @@
 trigger: none
 pr: none
 
-steps:
-    - script: | 
-        hello world
\ No newline at end of file
+
+jobs:
+  - job: Generate
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'ubuntu-16.04'
+    steps:
+    - template: templates/steps/install-reporting-tools.yml
+      parameters:
+        Options: --batch-mode
+        MavenOptions: $(MemoryOptions) $(LoggingOptions)
+
+    - task: Maven@3
+      displayName: 'Install azure-sdk-parent'
+      inputs:
+        mavenPomFile: parent/pom.xml
+        options: '$(DefaultOptions) -DskipTests -Dgpg.skip'
+        mavenOptions: '$(MemoryOptions)  $(LoggingOptions)'
+        javaHomeOption: 'JDKVersion'
+        jdkVersionOption: '1.11'
+        jdkArchitectureOption: 'x64'
+        publishJUnitResults: false
+        goals: 'install'
+
+    - task: Maven@3
+      displayName: 'Generate Maven project site, including JavaDocs, SpotBugs, and CheckStyle reports'
```

Comment:
@JonathanGiles, should we skip SpotBugs and Checkstyles here given master would be in an invalid state of they failed? Another option would be to continue running them in this step but turn failures off to generate, may want to do that whole sale for Javadocs and RevApi as well here.